aries-site/src/layouts/content/Example/ExampleResources.js

```diff
@@ -122,6 +123,11 @@ export const ExampleResources = ({
           </SyntaxHighlighter>
         </CollapsibleSection>
       )}
+      {caption && (
+        <Paragraph size="small" margin="none">
```

Comment:
I think we should add some bottom margin on the caption (small or medium?) -- haven't tried locally to know what would look better but right now it looks a bit tight

<img width="867" alt="Screen Shot 2021-11-02 at 11 30 00 AM" src="https://user-images.githubusercontent.com/12522275/139924135-1c309546-da17-4bb1-ac84-c0372c49af34.png">

Reply:
I can try this locally to see which one looks good